ArduPlane/tailsitter.cpp

```diff
@@ -244,15 +244,23 @@ bool QuadPlane::tailsitter_transition_fw_complete(void)
 {
     if (plane.fly_inverted()) {
         // transition immediately
+        gcs().send_text(MAV_SEVERITY_INFO, "Transition FW done, inverted flight");
         return true;
     }
     int32_t roll_cd = labs(ahrs_view->roll_sensor);
     if (roll_cd > 9000) {
         roll_cd = 18000 - roll_cd;
     }
-    if (labs(ahrs_view->pitch_sensor) > tailsitter.transition_angle*100 ||
-        roll_cd > tailsitter.transition_angle*100 ||
-        AP_HAL::millis() - transition_start_ms > uint32_t(transition_time_ms)) {
+    if (labs(ahrs_view->pitch_sensor) > tailsitter.transition_angle_fw*100) {
```

Comment:
these messages are a big improvement
but doesn't copter define "events" for such things?